src/coreclr-debug/activate.ts

```diff
@@ -30,6 +31,7 @@ export async function activate(thisExtension: vscode.Extension<CSharpExtensionEx
     }
 
     const factory = new DebugAdapterExecutableFactory(platformInformation, eventStream, thisExtension.packageJSON, thisExtension.extensionPath);
+    context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('coreclr', new CoreCLRConfigurationProvider(platformInformation)));
```

Comment:
>coreclr [](http://example.com/codeflow?start=80&length=7)

Since we are getting rid of 'csharp.listProcess', we need to do this for 'clr' also.